src/utils/tax-year-to-text.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function (question) {
+  const data = {};
+
+  if (question?.info?.currentTaxYear) {
```

Comment:
I think it would be neater if this function just takes the tax year string as its input rather than tying it to the shape of the question response - and maybe rename it `taxYearToRange`?